tutorials/preprocessing/90_eyetracking_data.py

```diff
@@ -101,123 +108,111 @@
-# blink artifacts biasing our analysis, so we have two options: We can either drop the
-# blink periods from our data during epoching, or we can interpolate the missing data
-# during the blink periods. For this tutorial, let's interpolate the missing samples:
-
-mne.preprocessing.eyetracking.interpolate_blinks(raw, buffer=0.05)
-# Let's plot our data again to see the result of the interpolation:
-raw.pick(["pupil_right"])  # Let's pick just the pupil channel
-raw.plot(events=events, event_id={"Flash": 3}, event_color="g")
-
-# %%
-# :func:`~mne.preprocessing.eyetracking.interpolate_blinks` performs a simple linear
-# interpolation of the pupil size data during blink periods. the ``buffer`` keyword
-# argument specifies the amount of time (in seconds) before and after the blinks to
-# include in the interpolation. This is helpful because the ``blink`` annotations
-# do not always capture the entire blink in the signal. We specified a value of ``.05``
-# seconds (50 ms), which is slightly more than the default value of ``.025``.
-
-# %%
-# Dealing with high frequency noise
-# ---------------------------------
-# From the plot above, we notice that there is some high frequency noise in the pupil
-# signal. We can remove this noise by low-pass filtering the data:
-
-# Apply a low pass filter to the pupil channel
-raw.filter(l_freq=None, h_freq=40, picks=["pupil_right"])
-
-# %%
-# Rejecting bad spans of data
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
-# Even after filtering the pupil data and interpolating the blink periods, we still see
-# some artifacts in the data (the large spikes) that we don't want to include in our
-# analysis. Let's epoch our data and then reject any epochs that might contain these
-# artifacts. We'll use :class:`mne.Epochs` to epoch our data, and pass in the
-# ``events`` array and ``event_dict`` that we created earlier. We'll also pass in the
-# ``reject`` keyword argument to reject any epochs that contain data that exceeds a
-# peak-to-peak signal amplitude threshold of ``1500`` in the ``"pupil"`` channel.
-# Note that this threshold is arbitrary, and should be adjusted based on the data.
-# We chose 1500 because eyelink reports pupil size in arbitrary units (AU), which
-# typically ranges from 800 to 3000 units. Our epochs already contains large
-# signal fluctuations due to the pupil response, so a threshold of 1500 is conservative
-# enough to reject epochs only with large artifacts.
+# blink artifacts biasing our analysis, so we have two options: Drop the blink periods
+# from our data during epoching, or interpolate the missing data during the blink
+# periods. For this tutorial, let's interpolate the blink samples:
+
+mne.preprocessing.eyetracking.interpolate_blinks(raw_et, buffer=(0.05, 0.2))
+
+# %%
+# .. note:: By default, :func:`~mne.preprocessing.eyetracking.interpolate_blinks`, will
+#           only interpolate blinks in ``"pupil"`` channels.
```

Comment:
```suggestion
# .. note:: By default, :func:`~mne.preprocessing.eyetracking.interpolate_blinks`, will
#           only interpolate blinks in ``"pupil"`` channels. Setting ``interpolate_gaze = False`` will also interpolate the blink periods of the ``"eyegaze"`` channels. Be aware, however, that eye movements can occur during blinks which makes the gaze data less suitable for interpolation.
```

Suggesting to add a bit more motivation here - I know it's also in the docstring, but maybe not everyone will land there ... WDYT @scott-huberty ?

Reply:
Yeah sorry I was thinking that linking directly to the function would allow users to see how to change the behavior if they were interested. 

 the interpolate gaze parameter there is backwards. Should be `interpolate_gaze=True` to also interpolate eye gaze channels. 

I'll commit this and make that fix locally and push back up!

Reply:
Ah of course 🤦 , my bad!